### ISSUE TITLE
return current status & basic bill info for a list of bills in the current biennium

### ISSUE THREAD

**Issue by rbriggs** [tannewt/wa-law.org#13]
This would enable groups who are doing bill tracking to construct a URL query that members could use to get a dump of their priority bills. Info could include meetings within a specified time period, and generate urls for sign in Pro/written comment. Bills could be sorted by status, so you could see just the ones that haven't passed yet, see what's passed or what's stuck in committee). Even better if it allows a group leader to construct named list, where the individual bills are saved with the account on the server side. Obviously that's more complicated.

**Comment by tannewt** [comment]
This would have to be done client side because the site itself is statically hosted.

**Comment by rbriggs** [comment]
Statically hosted? So you periodically download snapshots instead of making
queries on demand to waleg?

**Comment by tannewt** [comment]
Correct. I update this repo once an evening with the data. GitHub actions then converts the markdown files to html which is hosted by GitHub pages.
